packages/expo-updates/ios/EXUpdates/EXUpdatesAppDelegate.m

```diff
@@ -69,6 +61,18 @@ - (void)appController:(EXUpdatesAppController *)appController didStartWithSucces
 
 - (BOOL)shouldEnableAutoSetup
 {
+  // backward compatible if main AppDelegate already has expo-updates setup,
+  // we just skip in this case.
+  EXUpdatesAppController *controller = [EXUpdatesAppController sharedInstance];
+  if (controller.isStarted) {
+    return NO;
+  }
+
+  // if app is development build
+  if (EXAppDefines.APP_RCT_DEV) {
```

Comment:
would be nice to have a way to override this easily, in case you do want to make a debug build with expo-updates -- but maybe this is already possible through EXAppDefines?

Reply:
probably through the existing `EXUpdatesAutoSetup` plist key? right now it's boolean, we could change to support build types something like `EXUpdatesAutoSetupForBuildType: 'none' | 'release' | 'all'`

Reply:
I think that sounds good!